torchao/float8/float8_tensor.py

```diff
@@ -258,11 +263,19 @@ class Float8Tensor(torch.Tensor):
     * `_data`: the underlying e4m3 or e5m2 data
     * `_scale`: the scale used to scale the original fp32 tensor. We multiply
       by scale to go from fp32 range to fp8 range, and divide by scale to go
-      from fp8 range to fp32 range.
+      from fp8 range to fp32 range. Scale is guaranteed to have a shape compatible
+      with `_data`. For example:
+      - if scaling is tensorwise, `_scale` is a scalar tensor
+      - if scaling is axiswise and _data.shape is [3, 5], `_scale` could have
+        shape [1, 5] or [3, 1]. The dim of the non-one entry defines the scaling
```

Comment:
need to remove `The dim of the non-one entry defines the scaling axis` piece as it's not accurate for tensors which have multiple dimensions with size one